Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -394,6 +387,9 @@ define([
                     AttributeCompression.octDecodeInRange(octNormalUp[0], octNormalUp[1], 65535, instanceNormalUp);
                     AttributeCompression.octDecodeInRange(octNormalRight[0], octNormalRight[1], 65535, instanceNormalRight);
                     hasCustomOrientation = true;
+                } else if (hasTileTransform) {
+                    // Don't apply any orientation when the tile has a transform
+                    Matrix3.clone(Matrix3.IDENTITY, instanceRotation);
```

Comment:
@lasalvavida at some point can you regenerate the i3dm and cmpt tilesets with `EAST_NORTH_UP` set to true? Excluding the new ones added here.